helpers/sql/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry Sql Helpers
+
+Todo: Add a description.
+
+## How do I get started?
+
+Install the gem using:
+
+```console
+gem install opentelemetry-helpers-sql
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-helpers-sql` in your `Gemfile`.
```

Comment:
Since this is helper gem, and most users are likely be instrumentation authors/gems, should we include gemspec instructions?

Reply:
Likely no. I will rework the README to target the most appropriate audience.